app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('course.leaderboard.show.level_header')
```

Comment:
@weiqingtoh We prefer to user `t('.level_header')` here,  and use a symbol link in translation file which links to `course.leaderboard.show.level_header`